build.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-VERSION="0.2"
-MILESTONE=master
-RPM_RELEASE="0.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
+VERSION="1.1"
+MILESTONE=
+RPM_RELEASE="1.1$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
```

Comment:
RPM_RELEASE="1"

Reply:
in post release patch you should have:

RPM_RELEASE="0.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"